web-api/src/business/useCases/generateDocketRecordPdfInteractor.ts

```diff
@@ -80,10 +82,49 @@ export const generateDocketRecordPdfInteractor = async (
     });
 
   formattedCaseDetail.formattedDocketEntries =
-    formattedCaseDetail.formattedDocketEntries.map(docketEntry => ({
-      ...docketEntry,
-      numberOfPages: docketEntry.numberOfPages || 0,
-    }));
+    formattedCaseDetail.formattedDocketEntries.map(docketEntry => {
+      const formattedDocketEntry = {
+        ...docketEntry,
+        numberOfPages: docketEntry.numberOfPages || 0,
```

Comment:
Not a big deal, but do you think it would be worth adding this `|| 0` fallback to  `getFormattedCaseDetail`?